include/aspect/utilities.h

```diff
@@ -813,6 +813,12 @@ namespace aspect
          */
         std_cxx11::unique_ptr<aspect::Utilities::AsciiDataLookup<1> > lookup;
     };
+
+    template<int dim>
+    double compute_spd_factor(const SymmetricTensor<2,dim> &strain_rate,
```

Comment:
Can you write documentation what this function does, using `$\\varepsilon$`-style formulas as necessary?